### ISSUE TITLE
[FEATURE] Track/Detection Attributes Documentation 

### ISSUE THREAD

**Issue by BryonLewis** [VIAME/VIAME-Web#322]
Add in documentation for the track/detection attributes into the wiki.

Defining Attributes and difference between track and detection attributes

Setting up Attributes through the Settings interface
Process of creating attributes and the different options available
Number/Boolean/Text (including text lists)
Noting that they are a global setting on the server

Using/Applying attributes
Opening attributes to edit
Applying attributes and exporting them

Possibly some explanation of attributes when exporting in the CSV format.

**Comment by BryonLewis** [comment]
@subdavis and @mattdawkins - Attributes Wiki page for review
Attributes Wiki Page for Review - https://github.com/VIAME/VIAME-Web/wiki/Attributes
I created another page in the wiki.  I haven't hooked it up so it isn't visible to anyone unless you know the URL.  I just wanted it to be reviewed by some people so the understanding is correct before linking it up tot he rest of the documentation.  I tried to note the caveat about how attributes are stored globally on the server.  Once I'm given the go ahead by you guys I'll create a link to the core documentation to hopefully alleviate any further confusion.
